backend/app/mcp_server/tools/knowledge.py

```
@@ -0,0 +1,492 @@
+                    "description": spec.get("description", ""),
+                    "document_count": doc_count,
+                    "kb_type": spec.get("kbType", "notebook"),
+                    "created_at": kb.created_at.isoformat() if kb.created_at else None,
+                }
+            )
+
+        return {
+            "knowledge_bases": result,
+            "total": len(result),
+        }
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)